src/neo/Network/P2P/LocalNode.cs

```diff
@@ -129,6 +130,40 @@ internal static IPEndPoint GetIpEndPoint(string hostAndPort)
             return null;
         }
 
+        public bool CheckDuplicateNonce(IActorRef remoteActor, RemoteNode remoteNode)
```

Comment:
Maybe we can add some comments to the method.

Reply:
Method for checking duplicated duplicated Nonce.
Usually it occurs when xxxx...
If Nonce is the same we check if the Remote can be added to the known LocalAddresses.

If it is equal to the Nonce of other RemoteNode, we just return true.

Explain when we call `ConnectedPeers.TryUpdate(remoteActor, remoteNode.Listener, remote);`